test/karma/behaviorSpec.js

```diff
@@ -0,0 +1,522 @@
+      });
+      var viewWithBehavior = new ViewWithBehavior();
+      viewWithBehavior.attachTo(this.$app);
+      viewWithBehavior.$el.click();
+
+      var behaviorWithEvent = viewWithBehavior.getBehavior('behaviorWithEvent');
+      expect(savedContext).toBe(behaviorWithEvent);
+    });
+
+    it('can define events in both view hash and behavior hash', function() {
+      var ViewWithBehaviorAndEvent = TorsoView.extend({
+        behaviors: {
+          behaviorWithClickEvent: {
+            behavior: this.BehaviorWithClickEvent
+          }
+        },
+        events: {
+          'change': 'onChange'
+        },
+        onChange: jasmine.createSpy('onChange')
+      });
+
+      var viewWithBehaviorAndEvent = new ViewWithBehaviorAndEvent();
+      viewWithBehaviorAndEvent.attachTo(this.$app);
+      viewWithBehaviorAndEvent.$el.click();
+      viewWithBehaviorAndEvent.$el.change();
+      var behaviorWithClickEvent = viewWithBehaviorAndEvent.getBehavior('behaviorWithClickEvent');
+
+      expect(viewWithBehaviorAndEvent.onChange).toHaveBeenCalled();
+      expect(behaviorWithClickEvent.onClick).toHaveBeenCalled();
+    });
+
+    describe('if view.events is a method', function() {
+      it('can still fire events on view and behavior', function() {
+        var ViewWithBehaviorAndEvent = TorsoView.extend({
+          behaviors: {
+            behaviorWithClickEvent: {
+              behavior: this.BehaviorWithClickEvent
+            }
+          },
+          events: function() {
+            return {
+              'change': 'onChange'
+            };
+          },
+          onChange: jasmine.createSpy('onChange')
+        });
+
+        var viewWithBehaviorAndEvent = new ViewWithBehaviorAndEvent();
+        viewWithBehaviorAndEvent.attachTo(this.$app);
+        viewWithBehaviorAndEvent.$el.click();
+        viewWithBehaviorAndEvent.$el.change();
+        var behaviorWithClickEvent = viewWithBehaviorAndEvent.getBehavior('behaviorWithClickEvent');
+
+        expect(viewWithBehaviorAndEvent.onChange).toHaveBeenCalled();
+        expect(behaviorWithClickEvent.onClick).toHaveBeenCalled();
+      });
+
+      it('view.events is run in the context of the view', function() {
+        var savedContext;
+        var ViewWithBehaviorAndEvent = TorsoView.extend({
+          behaviors: {
+            behaviorWithClickEvent: {
+              behavior: this.BehaviorWithClickEvent
+            }
+          },
+          events: function() {
+            savedContext = this;
+            return {
+              'change': 'onChange'
+            };
+          },
+          onChange: jasmine.createSpy('onChange')
+        });
+
+        var viewWithBehaviorAndEvent = new ViewWithBehaviorAndEvent();
+
+        expect(savedContext).toBe(viewWithBehaviorAndEvent);
+      });
+    });
+
+    it('if duplicate events are defined in view and behavior event hash, only view handler runs', function() {
+      var ViewWithBehaviorAndEvent = TorsoView.extend({
```

Comment:
I'm leaning towards having both handlers run as well. The implementation right now is awkward because it's dependent on the event hash keys being the same. So, you could get two handlers to run on the same element if they used different selectors. If we wanted to allow both to run, we'd have to re-think how to implement registering the behavior events. Is this required for MVP?